pkg/ingress/controller/controller.go

```diff
@@ -58,7 +60,16 @@ func NewController(cfg *config.Config) (*Controller, error) {
 		podNamespace = "default"
 	}
 
-	conf.SetBaseUrl(cfg.APISIX.BaseURL)
+	client, err := apisix.NewForOptions(&apisix.ClusterOptions{
+		Name:     "",
```

Comment:
DefaultClusterName is “” ？

Reply:
It's consistent with the current design, i.e. group is "".